nox/sessions.py

```diff
@@ -368,6 +374,21 @@ def conda_install(
         if not args:
             raise ValueError("At least one argument required to install().")
 
+        if self._runner.global_config.no_install:
+            if (venv.venv_created):
+                logger.info(
+                    "Skipping {} conda installation, as --no-install is set.".format(
+                        args[0]
+                    )
```

Comment:
Unlike `session.run`, the first argument is not special. It could be a package or an option. I'd suggest to omit this from the message:

```suggestion
                    "Skipping conda installation, as --no-install is set."
```

Reply:
Thank you! I saw a package name in my tests and didn't reason much about what args[0] could actually be.